Microsoft.NET.Build.Containers/Registry.cs

```diff
@@ -269,7 +316,21 @@ public async Task Push(Image x, string name, string? tag, string baseName, Actio
                 await reg.Push(Layer.FromDescriptor(descriptor), name, logProgressMessage);
                 logProgressMessage($"Finished uploading layer {digest} to {reg.RegistryName}");
             }
-        }));
+        };
+
+        // Pushing to ECR uses a much larger chunk size. To avoid getting too many socket disconnects trying to do too many
```

Comment:
This makes me think that this should be pluggable or choosable - what if the task had a `LayerUploadMode` parameter that could be `Sequential` or `Parallel`, and different factors could influence the decision.

Not actionable for this PR, just something for us to noodle on.

Reply:
I suspect this logic needs to get more sophisticated for handling errors, throttling and errors. You could add parameters for users to tweak things but I would expect users just want it to work like the docker CLI which doesn't require users to have to think about this.